fix(debugger): correctly render frame error messages

michalkvasnicak opened the pull request:
## Change Summary

This PR fixes error message in frame preview.

## Merge Checklist

<!-- Check the completed and unnecessary tasks below -->

- [x] PR has a [Changeset](CONTRIBUTING.md)
- [x] PR includes documentation if necessary
- [x] PR updates the boilerplates if necessary

stephancill commented:
i don't think the updated behaviour is quite right.

below is a screenshot of an error response - it's not clear that it's an error (looks like a normal message)

<img width="849" alt="Screenshot 2024-05-05 at 16 47 25" src="https://github.com/framesjs/frames.js/assets/5469870/2a8eb2eb-5113-4599-961a-2d1ce8c93036">

here is a screenshot of a cast action response, there is no frame context associated with a cast action message response so it doesn't make sense to show it over a frame

<img width="851" alt="Screenshot 2024-05-05 at 16 54 58" src="https://github.com/framesjs/frames.js/assets/5469870/e4e3aebd-8c63-41e9-9839-1cdc174b0f08">

michalkvasnicak commented:
> i don't think the updated behaviour is quite right.
> 
> below is a screenshot of an error response - it's not clear that it's an error (looks like a normal message)
> 
> <img alt="Screenshot 2024-05-05 at 16 47 25" width="849" src="https://private-user-images.githubusercontent.com/5469870/328057941-2a8eb2eb-5113-4599-961a-2d1ce8c93036.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ5NzYxNzIsIm5iZiI6MTcxNDk3NTg3MiwicGF0aCI6Ii81NDY5ODcwLzMyODA1Nzk0MS0yYThlYjJlYi01MTEzLTQ1OTktOTYxYS0yZDFjZThjOTMwMzYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUwNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MDZUMDYxMTEyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MzEwMzEyMjFhNGUyNDQ0MjUwNzEzNjk0MGI5M2Y3YWM1NGUxY2IyY2FmMzk3NzAzZWUyMDdlYjc4MGM2ZGFmZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.D9TFQQ0Ub-9bxUI8T4lUbqIGk6_OLf01ig4wsG4LZYA">
> here is a screenshot of a cast action response, there is no frame context associated with a cast action message response so it doesn't make sense to show it over a frame
> 
> <img alt="Screenshot 2024-05-05 at 16 54 58" width="851" src="https://private-user-images.githubusercontent.com/5469870/328058261-e4e3aebd-8c63-41e9-9839-1cdc174b0f08.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ5NzYxNzIsIm5iZiI6MTcxNDk3NTg3MiwicGF0aCI6Ii81NDY5ODcwLzMyODA1ODI2MS1lNGUzYWViZC04YzYzLTQxZTktOTgzOS0xY2RjMTc0YjBmMDgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUwNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MDZUMDYxMTEyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MjYzOTA5NGMyOTBhYTI3MmRlYjEyODhhMWU1OWMzNDc3NmE2NTRmMWNkZTY0OTM3MjA2NDdkNjZjNDBjMDI2YiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.7Y1pehd-s9iqBPPpSWss4XmvkAySoiamji6gVj7-fCE">

I wasn't sure how should error message look like since the original implementation that was handling request errors wasn't looking like error either. I will change the styling to be of red color.

michalkvasnicak commented:
@stephancill I changed the text color to red, added inline version of error message if there is no image and also used the error message for the rest of errors we have there.

stephancill commented:
@michalkvasnicak i've added an error and info message variant to help disambiguate the two 
![Screenshot 2024-05-08 at 10 47 42](https://github.com/framesjs/frames.js/assets/5469870/fab18a92-94ad-42bb-84bb-100b82ddf2c7)
